db/migration/001_tweets_create.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+require 'sequel'
+
+Sequel.migration do
+  change do
+    create_table(:tweets) do
+      primary_key :id
+      String :tweet_id
+      String :create_time
```

Comment:
Can we change this to `datetime` data type? 
Also, I would prefer the name of this field to be `created_at`.